lms/djangoapps/instructor/views/data_access.py

```diff
@@ -0,0 +1,348 @@
+"""
+Methods associated with accessing models related to email list widget on instructor dashboard
+"""
+from courseware.models import StudentModule
+from courseware.models import GroupedQueries, GroupedQueriesSubqueries, GroupedTempQueriesSubqueries
+from courseware.models import QueriesSaved, QueriesStudents, QueriesTemporary
+from bulk_email.models import Optout
+from student.models import CourseEnrollment
+from django.contrib.auth.models import User
+from instructor.views.data_access_constants import INCLUSION_MAP, QUERY_TYPE, PROBLEM_FILTERS, SECTION_FILTERS
+from instructor.views.data_access_constants import INCLUSION, QUERYSTATUS
+from instructor.views.data_access_constants import REVERSE_INCLUSION_MAP, Query
+from instructor.views.data_access_constants import DATABASE_FIELDS
+from django.db.models import Q
+from collections import defaultdict
+import random
+import datetime
+from celery import task
+
+
+def delete_saved_query(query_to_delete):
```

Comment:
can I ask that we be a bit more explicit in the type of the argument here?  I _think_ that `query_to_delete` is just a primary key, and is therefore an `int`?  In that case, can we call it `query_id_to_delete`?  Just b/c sometimes querysets are lazy and can accept both a python object and an id, but let's be clear on which one we're supplying.